Makefile

```diff
@@ -73,6 +73,11 @@ ifeq ($(UNAME_O),Cygwin)
 	LIBS += -liconv
 	EXT = .exe
 endif
+ifeq ($(UNAME_O),Msys)
+	CFLAGS += -I/mingw64/include -I/mingw64/lib/libzip/include
+	LIBS += -liconv -llibzip -L/mingw64/lib -lzip -lz -L/mingw64/lib -L/mingw64/bin
```

Comment:
There is no need to pass `-I/mingw64/include` or `-L/mingw64/lib`, and `-L/mingw64/bin` is actually incorrect ;-)